.rubocop.yml

```diff
@@ -5,6 +5,8 @@ AllCops:
   RunRailsCops: true
 Metrics/LineLength:
   Enabled: false
+Metrics/ClassLength:
+  Enabled: false
```

Comment:
Oh, totally, we are breaking a _lot_ of our enforced rules:

https://codeclimate.com/github/18F/C2/issues

Luckily Code Climate only fails the build for style rules in lines that you change. I've only been disabling rules that I disagree with or don't think are important.